packages/firebase_crashlytics/android/build.gradle

```diff
@@ -0,0 +1,42 @@
+group 'io.flutter.plugins.firebase.crashlytics.firebasecrashlytics'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    repositories {
+        google()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.2.1'
+    }
+}
+
+rootProject.allprojects {
+    repositories {
+        google()
+        jcenter()
+        maven {
+            url 'https://maven.fabric.io/public'
+        }
+    }
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        minSdkVersion 16
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+}
+
+dependencies {
+    implementation 'com.google.firebase:firebase-core:16.0.6'
+    implementation 'com.crashlytics.sdk.android:crashlytics:2.9.8'
```

Comment:
No Crashlytics NDK support to capture low-level errors such as in Flutter Engine?

**Dependency**: `implementation 'com.crashlytics.sdk.android:crashlytics-ndk:2.0.5'`


Users `build.gradle` (documentation step only): 

```groovy
crashlytics {
  enableNdk true
}
```

See: https://docs.fabric.io/android/crashlytics/ndk.html

Reply:
I feel like that should be separate dependency

Reply:
@Salakar thanks for the suggestion, we can definitely add this in a future version.